packages/trip-details/src/TripDetails.story.tsx

```diff
@@ -348,3 +348,14 @@ export const FareComponentsItinerary = makeStory({
 });
 
 export const OTP2FlexItinerary = makeStory({ itinerary: flexItinerary });
+
+export const FareLegTableStory = (): ReactElement => {
+  return (
+    <FareLegTable
+      layout={fareByLegLayout}
+      legs={walkInterlinedTransitItinerary.legs}
+      transitFareDetails={walkInterlinedTransitItinerary.fare?.details}
+      transitFares={walkInterlinedTransitItinerary.fare?.fare}
```

Comment:
Oy vey...

Reply:
I have no idea why I did it like this... It definitely seems like it could have just accepted the itinerary. I could make a bug for that